src/backend/metadata/search.ts

```diff
@@ -24,6 +24,12 @@ export async function searchForMedia(query: MWQuery): Promise<MediaItem[]> {
     return formatTMDBMetaToMediaItem(formattedResult);
   });
 
-  cache.set(query, results, 3600); // cache results for 1 hour
-  return results;
+  const movieWithposters = results.filter((movie) => movie.poster);
+  const movieWithoutposters = results.filter((movie) => !movie.poster);
+
+  const sortedresult = movieWithposters.concat(movieWithoutposters);
```

Comment:
Minor stylistic nit-pick

```suggestion
  const movieWithPosters = results.filter((movie) => movie.poster);
  const movieWithoutPosters = results.filter((movie) => !movie.poster);

  const sortedresult = movieWithPosters.concat(movieWithoutPosters);
```